subprojects/docs/src/docs/userguide/migration/upgrading_version_6.adoc

```diff
@@ -70,6 +70,13 @@ Please refer to the <<publishing_maven.adoc#publishing_maven, documentation of t
 The deprecated Play plugins have been removed.
 An external replacement, the link:https://gradle.github.io/playframework[Play Framework plugin], is available from the plugin portal.
 
+=== Removal of deprecated JVM plugins
+
+These unmaintained alternative JVM plugins have been removed:
+`java-lang`, `scala-lang`, `junit-test-suite`, `jvm-component`, `jvm-resources`
+
+Please use the stable <<java_library_plugin.adoc#,Java Library>> and <<scala_plugin.adoc#,Scala>> plugins instead.
+
```

Comment:
I added all the "JVM lang" plugins here already. Then there is nothing to do in the follow up PR.